### PR TITLE
Clean up duplicates that are already in ckanext-blob-storage

### LINKED ISSUE #8: Duplicate templates between ckanext-blob-storage and ckanext-datapub
There are a few templates that are the same or very close between ckanext-blob-storage and this extension, and it's just confusing. 

If ckanext-blob-storage will always depend on datapub, and has no UI at all, the templates should be removed from there. But I don't like this solution cause it creates cyclical dependencies between these two extensions, and I think ckanext-blob-storage should have some kind of basic UI. So I suggest assuming the base templates exist in ckanext-blob-storage (as this extension has a dependency on it anyway) and removing the duplicates from here. 

You can then override only `templates/blob_storage/snippets/upload_module.html` to point to the right JS files, no need to also override `templates/package/*`.

### PR DESCRIPTION
**Pull request by shevron**
This PR resolves #8 by removing anything that already exists in ckanext-blob-storage, under the assumption that this extension will always depend on it. We only override the uploader_module template here. 

The `sync.sh` script was also cleaned up a bit.

### CODE REVIEW

File: ckanext/datapub/templates/external_storage/snippets/upload_module.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
 
 <div id="ResourceEditor"
-     data-dataset-id="{{ pkg_name }}"
+     data-dataset-id="{{ pkg_id }}"
      data-api="{{ base_url }}"
-     data-lfs="{{ h.extstorage_lfs_url() }}"
+     data-lfs="{{ h.extstorage_server_url() }}"
      data-auth-token="{{ api_key }}"
-     data-organization-id="{{ h.extstorage_organization_name(pkg_name) }}"
+     data-organization-id="{{ h.extstorage_storage_namespace() }}"
```
Comment (mariorodeghiero) [inline ckanext/datapub/templates/external_storage/snippets/upload_module.html:12]: @shevron I found the `extstorage_server_url` helper in [ckanext-blob-storage](https://github.com/datopian/ckanext-blob-storage/blob/master/ckanext/external_storage/plugin.py#L31) but I didn't find `extstorage_storage_namespace` where is it coming from?

Comment (shevron) [reply ckanext/datapub/templates/external_storage/snippets/upload_module.html:12]: This is referring to a new branch of ckanext-blob-storage, which has not been merged yet. I want to merge both PRs together.

Comment (mariorodeghiero) [reply ckanext/datapub/templates/external_storage/snippets/upload_module.html:12]: nice, for me looks good.

**Review by mariorodeghiero — APPROVED** [review]
Looks good to me. 

@anuveyatsu I am not sure if is there a problem for NHS?

**Comment by anuveyatsu** [comment]
@mariorodeghiero nhs team has created a separate branch here so implicitly the master branch is just for gdx now. I'm not sure which approach we should follow here, eg, you can see that there is another pr that is gdx specific #7

**Review by anuveyatsu — APPROVED** [review]
LGTM

**Comment by mariorodeghiero** [comment]
@shevron I deleted the branch to keep it clean.

@anuveyatsu yes I am not sure too.

**Pull request merged**